exchanges/okx/okx_websocket.go

```diff
@@ -1232,38 +1232,41 @@ func (ok *Okx) wsProcessTickers(data []byte) error {
 func (ok *Okx) GenerateDefaultSubscriptions() ([]stream.ChannelSubscription, error) {
 	var subscriptions []stream.ChannelSubscription
 	assets := ok.GetAssetTypes(true)
+	subs := make([]string, len(defaultSubscribedChannels))
+	copy(subs, defaultSubscribedChannels)
 	if ok.Websocket.CanUseAuthenticatedEndpoints() {
-		defaultSubscribedChannels = append(defaultSubscribedChannels,
+		subs = append(subs,
```

Comment:
Just noticed this: https://github.com/thrasher-corp/gocryptotrader/actions/runs/5216358886/jobs/9414995924?pr=1221#step:4:2832